bin/moveAndCheckSamplesheets.sh

```diff
@@ -239,11 +239,29 @@ else
 		if [[ -n "${_sampleSheetColumnOffsets["SentrixBarcode_A"]+isset}" ]] 
 		then
 			log4Bash 'INFO' "${LINENO}" "${FUNCNAME:-main}" '0' "This is a GAP samplesheet. There is no samplesheetCheck at this moment."
+			#
+			# Make sure
+			#  1. The last line ends with a line end character.
+			#  2. We have the right line end character: convert any carriage return (\r) to newline (\n).
+			#  3. We remove empty lines.
+			#
+			printf '\n'     >> "${samplesheet}.converted"
+			sed 's/\r/\n/g' "${samplesheet}.converted" > "${samplesheet}.converted.tmp"
+			sed '/^\s*$/d'  "${samplesheet}.converted.tmp" > "${samplesheet}.converted.tmp2"
+			rm "${samplesheet}.converted.tmp"
+			mv "${samplesheet}.converted.tmp2" "${samplesheet}.converted"
+			mv "${samplesheet}.converted" "${samplesheet}"
```

Comment:
This will work, but now we have the same code 3 times, which is a bit redundant. Why not moving it out of the `if` statement to ~line 228, so it goes before all code trying to parse the samplesheet.
* With `-i` for _in place_ replacement with `sed`, the code can be shorter with one tmp file less.
* Adding `${SAMPLESHEET_SEP}` to the second sed will also remove "empty" lines, that only contain comma's, which is something Excel, sometimes produces.
So I suggest we use this:
```
			#
			# Make sure
			#  1. The last line ends with a line end character.
			#  2. We have the right line end character: convert any carriage return (\r) to newline (\n).
			#  3. We remove empty lines.
			#
			printf '\n'     >> "${samplesheet}.converted"
			sed -i 's/\r/\n/g' "${samplesheet}.converted"
			sed -i "/^[\s${SAMPLESHEET_SEP}]*$/d" "${samplesheet}.converted"
			mv "${samplesheet}.converted.tmp2" "${samplesheet}.converted"
			mv "${samplesheet}.converted" "${samplesheet}"
```